packages/sanes-chrome-extension/src/extension/background/actions/createPersona/requestHandler/index.unit.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { Request, RequestHandler } from './index';
+
+describe('RequestHandler', () => {
+  const REQUEST_ONE: Request = {
+    id: 1999,
+    type: 'getIdentities',
+    reason: 'Reason 1',
+    sender: 'Sender 1',
+    accept: () => {},
+    reject: () => {},
+  };
+
+  const REQUEST_TWO: Request = {
+    id: 2999,
+    type: 'signAndPost',
+    reason: 'Reason 2',
+    sender: 'Sender 2',
+    accept: () => {},
+    reject: () => {},
+  };
+
+  const REQUEST_THREE: Request = {
+    id: 3999,
```

Comment:
There is no need to declare `id`, it should be added automatically when `add`. In fact this is something we need to unit test.

` add | add | solve | solve | add | reject | add` <--- the last one should be 3 (counter starts on 0) `

Reply:
I'm aware the `id`s are inserted by `add`, it is tested on the last test. I put those `id`s to comply with the Request interface so that the `add` method accepts this objects as arguments.

Is there any way to remove the id property from these objects and `add` still accepts them as arguments? Because as soon as I remove `id` it can no longer be a `Request` object.

Reply:
You can use: ```Omit<Request, 'id'>``` in TS 3.5 it will be added by default, right now just use  `import { Omit } from '@material-ui/core';`